packages/plugins/rubygem-smart_proxy_remote_execution_ssh/rubygem-smart_proxy_remote_execution_ssh.spec

```diff
@@ -8,14 +8,19 @@
 Summary: SSH remote execution provider for Foreman smart proxy
 Name: rubygem-%{gem_name}
 Version: 0.2.1
-Release: 1%{?dist}
+Release: 2%{?dist}
 Group: Applications/System
 License: GPLv3
 URL: https://github.com/theforeman/smart_proxy_remote_execution_ssh
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
+%if 0%{?rhel} == 7
 Requires: %{?rhel:tfm-}rubygem(smart_proxy_dynflow_core) >= 0.1.5
```

Comment:
the `?rhel:` seems redundant now and we can just write `tfm-rubygem(…)`?

Reply:
Looks like there's a precedent for that:
https://github.com/theforeman/foreman-packaging/blob/403a3e050607433634b29e13a8b6e3e8b8ab2a61/packages/plugins/rubygem-smart_proxy_dynflow/rubygem-smart_proxy_dynflow.spec#L20-L26